### ISSUE TITLE
2024 Decide how to boost computer graphics skill 

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#1010]
Step toward https://github.com/kinu-garage/essay_in_idleness/issues/150. Can be potentially included in https://github.com/kinu-garage/essay_in_idleness/issues/243
MOOC makes learning CG much more comfortable I think.

**Comment by 130s** [comment]
- [edx.org](https://www.edx.org/learn/computer-graphics/the-university-of-california-san-diego-computer-graphics?irclickid=R48Q-oXHrxyPR2q0y7SIT2hkUkHURb2NzT6IVc0&utm_source=affiliate&utm_medium=CoToNet%20-%20Tecnologias%20de%20Informa%C3%A7%C3%A3o%2C%20Unipessoal%20Lda&utm_campaign=Online%20Tracking%20Link_&utm_content=ONLINE_TRACKING_LINK&irgwc=1) by UCSD professor $99 or free
- [udemy](https://www.udemy.com/course/the-complete-computer-graphics-course/?couponCode=2021PM20#instructor-1) by a post-master engineer. Looks good although $15 and only 3 reviews as well as no coding experience being required make me nervous.

**Comment by 130s** [comment]
Speaking of mooc platform btw, this artcile https://careerkarma.com/wiki/udemy-vs-edx/ interestingly flags Udemy below edx.

**Comment by 130s** [comment]
Started on edx.

**Comment by 130s** [comment]
There's a follow-on course https://discussions.edx.org/course-v1:UCSanDiegoX+CSE167x+2T2018/posts/5f385d6526ea960a2efd9211
> I am very excited to let you know that we are officially announcing a follow-on course, CSE 168 Computer Graphics II: Rendering, to start on Sep 1 and hosted on UCSD online. Please create an account and register at http://online.ucsd.edu If you can't find it, a direct link to the course enrollment page is https://online.ucsd.edu/courses/course-v1:CSE+168X+2020-SP/about
> 
> (Like 167, the course is self-paced and will be active for at least the next 6 months and very likely extended, so please sign up even if you don't immediately have time for it).
> 
> CSE 168 is a course on modern physically-based rendering, and follows on from where CSE 167 leaves off. You are well qualified for it if you passed CSE 167. You do not need to have done the final raytracer assignment, since it is anyway the first homework in CSE 168. Strictly speaking, CSE 168 does not require OpenGL, and therefore you may be able to do it without having fully completed CSE 167, although realistically you should have passed CSE 167 before attempting CSE 168. However, you are always welcome to register and browse assignments and videos.
> 
> Feel free to also let any colleagues or friends know. The course is now openly available and announced, so we welcome all comers. We are still working out the logistics in terms of payments, but it is likely the course (at least for now) will be completely free for access to lectures and assignments, with only a nominal $49 fee for those who want the statement of accomplishment upon completion of the course. The course materials are exactly the same as what we used for our UCSD CSE 168 this spring, and in fact the goal is ultimately to have UCSD students also use the same infrastructure.
> 
> The course will be run directly by UCSD, using its new UCSD online platform, which is built on OpenedX. While it uses the same edX infrastructure that you are familiar with, you will need to create a new account and register for the course navigating to https://online.ucsd.edu If you can't find it, a direct link to the course enrollment page is https://online.ucsd.edu/courses/course-v1:CSE+168X+2020-SP/about
> 
> Please do not hesitate to get in touch with me directly at ravir@cs.ucsd.edu regarding any concerns or questions.

**Comment by 130s** [comment]
> Speaking of mooc platform btw, this artcile https://careerkarma.com/wiki/udemy-vs-edx/ interestingly flags Udemy below edx.

Just ran into slew of articles that report the bankruptsy of the parent? company 2U e.g. https://www.thecrimson.com/article/2024/7/27/2u-bankruptcy-harvard-mit-edx/
I found their website may not be maintained well, log in feature has caused me errors, customer support is responsive but 2nd time I requested help they aren't quick. Social sign-in feature w/FB seems broken (which happens a lot on other websites though) and Google social feat. does unclear stuff.

Better to finish taking the courses I want  with the fear that the service becomes unusable. Hope that won't happen.
